<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicRoom</class>
 <widget class="QMainWindow" name="MusicRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Room</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="TreeWidget_Category">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Musics</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Folder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PlayLists</string>
     </property>
     <item>
      <property name="text">
       <string>Happy</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Queue</string>
     </property>
    </item>
   </widget>
   <widget class="QListView" name="ListView_AudioTracks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Audio Tracks:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_PlayPause">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Next">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Prev">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Shuffle">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Suffle: Off</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Folder Selection</string>
    </property>
    <widget class="QPushButton" name="PushButton_Browse">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit_Path">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>381</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>D:/Music/Playlist 1</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="PushButton_Repeat">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RepeatMode: No-Repeat</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_MusicSlider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Timer">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
